src/tokens/safe.tokenlist.json

```diff
@@ -47,6 +47,14 @@
       "decimals": 18,
       "chainId": 100,
       "logoURI": "https://safe-transaction-assets.safe.global/tokens/logos/0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB.png"
+    },
+    {
+      "symbol": "WMATIC",
+      "name": "Wrapped Matic",
+      "address": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
```

Comment:
The other addresses are in lowercase. I am not sure if that was a requirement or not but I'd make it lowercase just in case.

Reply:
Lower-cased it. After we merge this, will the token be updated automatically in the UI?

Reply:
The job to update the token list can be triggered manually by Uxio. Otherwise it will happen once a day.